### ISSUE TITLE
Textarea - Accessibility Violation with labelHelp 

### ISSUE THREAD

**Issue by divyajindel** [Sage/carbon#4676]
### Current behaviour

ARIA hidden element must not contain focusable elements

### Expected behaviour

There are no accessibility violations.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/textarea--with-label-help

### Carbon Version

latest

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS

### Anything else we should know?

![Uploading Screenshot 2022-01-05 at 21.54.00.png…]()

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.
